pkg/plugins/golang/v3/api.go

```diff
@@ -57,6 +57,7 @@ type createAPISubcommand struct {
 	pattern string
 
 	resource *resource.Options
+	api      config.API
```

Comment:
Options already contains a `config.API`
```suggestion
	resource *resource.Options
```